### PR TITLE
Update container VM image size and bridge network signposts

### LINKED ISSUE #59: Add signpost text explaining the implications of the base image size
The text below the input warning the user that the value cannot be changed is helpful. Adding signpost text to explain the purpose of the value may further assist users.

Again, explaining this in terms familiar to a VI Admin may be valuable.

Additionally, it may be helpful to clarify that the setting is the base _container VM_ image size.

### LINKED ISSUE #60: Add signpost text to the bridge network settings that explicitly notes this should be an isolated L2 broadcast domain
Additionally, we could say that it should be a VXLAN or should be configured with a VLAN ID.

### PR DESCRIPTION
**Pull request by cristianfalcone**
Fixes #59 
Fixes #60

### CODE REVIEW

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.scss [hunk 1]
```diff
@@ -8,7 +8,14 @@
...
+:host ::ng-deep clr-signpost {
+  .signpost-action {
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+}
+
+.bridge-network-signpost {
+  min-width: 600px;
+  max-width: 600px;
```
Comment (jak-atx) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.scss]: Why is this necessary here to set min and max to the same? Can we not set the width only?

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html [hunk 2]
```diff
@@ -56,7 +56,7 @@
       <div class="col-xs-3">
 
         <label>
-          Base image size
+          Maximum Container VM image size
```
Comment (jak-atx) [inline h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html]: This is wrapping to 2 lines and hampers readability. Checked with Louis and rather than adjusting layout he suggested to change the text to read "Max Container VM image size" to help it some.

Comment (jak-atx) [reply h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html]: https://vmware.invisionapp.com/share/87DSS6N2U#/screens

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.scss (hunk 1 above)
Comment (cristianfalcone) [reply h5c/vic/src/vic-webapp/src/app/create-vch-wizard/networks/networks.scss]: Since the signpost is inside a col-xs-1, it is using the min-width which results in a thin and tall signpost popup due to the long text inside. Setting width to signpost max-width now.

File: h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html (hunk 2 above)
Comment (cristianfalcone) [reply h5c/vic/src/vic-webapp/src/app/create-vch-wizard/storage-capacity/storage-capacity.html]: Changing Maximum to Max, leaves the word "size" alone, in the second line. Adding a <br>, and a new line-height now, so it looks like the mockups.
